app/models/validators/asylum_support_decision_validator.rb

```diff
@@ -14,6 +15,16 @@ class AsylumSupportDecisionValidator < SimpleDelegator
   validates :tribunal_decision_judges, presence: true
   validates :tribunal_decision_landmark, presence: true
   validates :tribunal_decision_reference_number, presence: true
-  validates :tribunal_decision_sub_category, presence: true
+  validates :tribunal_decision_sub_category, tribunal_decision_sub_category: true
```

Comment:
I can make them separate validators if you think that works. Thought it might be better to leave it as a single validator so all sub-category validation logic is in one file. It gets tricky as the sub-category presence validation is contingent on the value of the parent category, so in a way is part of the same broader validation. Would you still like me to split this into separate validator classes?